pkg/controllers/dynakube/istio/conditions.go

```diff
@@ -0,0 +1,30 @@
+package istio
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getConditionTypeName(component string) string {
+	return "IstioServiceConfigurationFor" + component
```

Comment:
this feels a bit too verbose, my suggestions for alternatives:
- `<component>IstioConfig`
- `IstioFor<component>`

Reply:
fixed in #e131fa0d